inc/commondbtm.class.php

```diff
@@ -3765,8 +3767,13 @@ function deleteByCriteria($crit=array(), $force=0) {
       if (is_array($crit) && (count($crit) > 0)) {
          $crit['FIELDS'] = 'id';
          $ok = true;
+         $history = 1;
+         if(isset($crit['_no_history'])){
```

Comment:
instead of a crit (which don't corresponds to your need, i'll prefer a new function parameter (set to true at default).